rust/common/identifier.rs

```diff
@@ -23,10 +23,7 @@ use std::sync::OnceLock;
 
 use regex::{Regex, RegexBuilder};
 
-pub fn is_valid_identifier(identifier: &str) -> bool {
-    static REGEX: OnceLock<Regex> = OnceLock::new();
-    let regex = REGEX.get_or_init(|| {
-        let identifier_start = "A-Za-z\
+const IDENTIFIER_CHAR: &str = "A-Za-z\
```

Comment:
Refactored constant strings out here, and only compute the regexes once in the `OnceLock`.